### DISCUSSION TITLE
specify the path “/tm/login”

### DISCUSSION THREAD

**Discussion by HHZZUUJH** [typemill/typemill#495] [Q&A]
Hello and best regards,

I would very much like to use this application. I have installed typemill on a Synology NAS as a Docker container.

Since port 443 is already in use, I have specified a different port for HTTPS. 

To log in, I have to specify the path “/tm/login” in the URL after the port.

However, the Synology reverse proxy does not process paths. Only top-level domain names and ports. 

Therefore, I cannot access typemill via HTTPS from the Internet. 

Is it possible to access the login page of Typemill directly without the path “/tm/login”?

kind regards

**Comment by trendschau** [comment]
Sorry, but I am not familiar with a setup like this. The path /tm/... cannot be changed right now and the login works the traditional way with post request to another route. There is an api mode (not finished), where you can authenticate each request with basic auth, but I guess this will not help you in this scenario. 

If you find another solution, it would be nice to get a feedback in case other users want to run Typemill with a similar setup.
